library/general/httpsonly/template.yaml

```diff
@@ -3,8 +3,15 @@ kind: ConstraintTemplate
 metadata:
   name: k8shttpsonly
   annotations:
-    description: Requires Ingress resources to be HTTPS only; TLS configuration should
-      be set and `kubernetes.io/ingress.allow-http` annotation equals false.
+    description: >-
+      Requires Ingress resources to be HTTPS only.
+
+      Ingress resources must:
+      - include a valid TLS configuration
```

Comment:
Will this list format correctly after YAML wrapping?

Reply:
No, it will not. With `>` newlines are replaced with spaces so this list will become one line.

Reply:
This one isn't long enough to warrant complex formatting, but it looks like we can use `|-` to keep things like newlines (we do this with rego).  It'll be worth some testing in the future to see if we'd like to pursue more formatting in our descriptions.

Reply:
+1 for `|-`

Do we prever `>-` for some reason?

Reply:
Wrapping long lines in the YAML that we don't want to be wrapped universally.

Otherwise we are coupling the GitHub code review line width (which is rather short), with HTTP text wrapping (where the flow should be determined by the screen width, see [responsive web design](https://en.wikipedia.org/wiki/Responsive_web_design))